ops/testing.py

```diff
@@ -407,16 +407,53 @@ def add_relation(self, relation_name: str, remote_app: str) -> int:
         self._emit_relation_created(relation_name, rel_id, remote_app)
         return rel_id
 
+    def remove_relation(self, relation_name: str, remote_app: str, rel_id=None) -> None:
```

Comment:
rel_id -> relation_id: str=None

makes the parameter name consistent with other methods.

Reply:
OK will change parameter name to "relation_id". When writing the charm code isn't the relation name the more developer friendly interface ? If so wouldn't it be appropriate to require a relation id only if the relation name is ambiguous ?

Reply:
Changed parameter name to `relation_id`. Let me know if the above rationale is acceptable. If not I can make `relation_id` a mandatory argument and remove `relation_name` as it will be redundant, since there is a internal map `_TestingModelBackend._relation_names` from relation ids to relation names.